docs/dev/api_integration.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Create an API Integration
+type: documentation
+---
+
+## Overview
+
+You can use [Datadog API endpoints][1] to programmatically send data to Datadog, manage your account, and visualize data collected from an API integration. 
+
+API integrations send the following types of data to Datadog:
+
+- [Metrics][12]
+- [Logs & Log Pipelines][13]
+- [Events][14]
+- [Service Checks][15]
+- [Traces][16]
+- [Incidents][17]
+- [Security Events][18]
+
+This page provides instructions for creating an API integration in the `integrations-extras` repository. For more information about why you would want to create an API-based integration, see [Creating your own solution][2].
```

Comment:
```suggestion
This page provides instructions for creating an API integration in the `integrations-extras` repository. For more information about why you would want to create an API-based integration, see [Creating your own solution][2]. Since API integrations do not use the Datadog Agent to collect data, you need to create an informational tile-only listing once your development work is complete. 
```